common/store/store.ts

```diff
@@ -65,10 +65,10 @@ const configureStore = () => {
     ...rehydrateConfigAndCustomTokenState()
   };
 
-  store = createStore(RootReducer, persistedInitialState, middleware);
+  store = createStore<AppState>(RootReducer, persistedInitialState as any, middleware);
```

Comment:
I think if persistedInitialState is typed as a partial AppState, this should work.

Reply:
this doesn't work because `createStore` expects a full AppState, when it should be a DeepPartial of AppState

Reply:
Ah, I knew this was familiar. They updated `index.d.ts` in `redux` to allow for deep partials (https://github.com/reactjs/redux/commit/cbdca6215e1891e97120ac05c971618182455e54), but they haven't updated the npm module yet, so it's got old typings. We'll just have to remember to do this the next time redux is updated.